lib/parsers/package-lock-parser.ts

```diff
@@ -120,8 +120,13 @@ export class PackageLockParser implements LockfileParser {
 
     // number of dependencies including root one
     let treeSize = 1;
-
+    const treeSizeLimit = 6.0e6;
+  
     for (const dep of topLevelDeps) {
+      // On Call we decided that the tree size limit will be of 6 millions.
```

Comment:
Probably this shouldn't be there. Just give brief explanation, please

Reply:
Shall we have a test for this?

Reply:
We can have a side pr with fixtures where we can simulate the 6m+ test

Reply:
Why can't it be here?

Reply:
I wanted to apply that in side pr, due of the current needs. But no worries I will update with test.

Reply:
Due of current priority, we do not have time to create a fixture for that now.